### DISCUSSION TITLE
Filter out shovelware

### DISCUSSION THREAD

**Discussion by cosmo0** [benbarden/switchscores#90] [Ideas]
It's a fact that the Switch eShop has a lot of shovelware.

It'd be great if users could filter them out (and even better if it was done by default).

Some companies are specialized in shovelware and could be hidden for everyone:

- Ultimate Games : https://www.switchscores.com/partners/games-company/ultimate-games
- Sabec : https://www.switchscores.com/partners/games-company/sabec
- Baltoro : https://www.switchscores.com/partners/games-company/baltoro-games
- Pix Arts : https://www.switchscores.com/partners/games-company/pix-arts
- EnjoyUp : https://www.switchscores.com/partners/games-company/enjoyup-games
- Piotr Skalski : https://www.switchscores.com/partners/games-company/piotr-skalski
- Digital Game Group : https://www.switchscores.com/partners/games-company/digital-game-group
- SuperPowerUp : https://www.switchscores.com/partners/games-company/super-powerup-games
- StarSign : https://www.switchscores.com/partners/games-company/starsign
- Nestor Yavorskyy : https://www.switchscores.com/partners/games-company/nestor-yavorskyy
- Kistler Benjamin: https://www.switchscores.com/partners/games-company/kistler-benjamin
- Alcyone Studio: https://www.switchscores.com/partners/games-company/alcyone-studio

(I missed a lot of them, it's just a quick list)

Some companies just make mediocre games.
Sometimes it's not companies but single games.

I think it would help a lot if people could hide games and companies from lists and results.

**Comment by benbarden** [comment]
We could probably do different things on different pages. Some ideas:

- We could have some kind of "shovelware" message and icon that appears on partner pages, game pages, and listings (instead of the 1/3 reviews or the rank). The alert would need to link to a page with an explanation of what this means, and the opportunity to get in touch. It's possible to make a mistake with this, and if a publisher saw their games listed as shovelware, they might want to appeal. That would be interesting!
- Similar to de-listed games, shovelware would not be included in game ranks - though to be honest, most won't have ranks anyway. So, unranked games pages would not include shovelware, as it means we aren't asking partners or members to review them.
- Category pages could have an extra tab for shovelware - so the games don't appear in the main list, but aren't removed completely.

**Reply by cosmo0** [reply]
The first two are fine.

The extra tab - I don't think people would go "hey I want to buy a shovelware what do you have". Just hiding them would be enough, to de-clutter the listings.

**Comment by GrimTalin** [comment]
Just going to throw my two cents here as a dev. I think labeling something as shovelware is too subjective. If a user can just filter out unranked games (less than 3 reviews) or games without any review, I think that would probably be enough, without labeling a title in any demeaning way. Someone's shovelware might be a decent or even good game for someone else, I've seen it happen plenty of times. My Elena Temple got called shovelware by some too, but it was plenty loved by a lot of people. So, like I said, I think it's too subjective and would recommend to avoid the term, much as we'd all like to catalogue some titles on the Switch as such.

**Reply by cosmo0** [reply]
I tend to agree with you. Personnaly I'd prefer if it was a user setting, to hide publishers they don't like, or specific titles (like titles that come up in sales too often). It could even be used to hide non-shovelware, like tags ("I don't like pixel art") or such.

But you know. Sabec. Just hide Sabec? 🤔

**Reply by benbarden** [reply]
Good thoughts. Something I'd like to do, in an ideal world, would be to get most of the games on the site to be reviewed - better still, ranked. To help out, you'd have to see all of the unranked games. Games from e.g. Sabec and Pix Arts especially are such poor quality that I wouldn't want people to spend a lot of time reviewing the games, let alone pay for them. In some cases these are demos taken from game maker tools so shouldn't be on the eShop at all ... 

Though as you say, it is indeed subjective, and if some of the games had reviews the scores would speak for themselves.

**Comment by benbarden** [comment]
I added a Low Quality filter that addresses this.
